### DISCUSSION TITLE
Global Speed playback

### DISCUSSION THREAD

**Discussion by bcdalejos** [PaulWoitaschek/Voice#2485] [Ideas]
This speed will apply to all books

**Comment by dalaeance** [comment]
Seconded. It's only a slight inconvenience, but it'd be nice if:
- We could set a default speed in the settings and/or
- The app remembered our chosen speed between files

A bonus might be if we could have the default, but have a way to set speeds for individual files that may still be too slow, without affecting the default / set speed for others.

But, for now, I feel simply doing one, or both, of the bulleted options would be enough. The least amount of work to benefit the most amount of people.

**Comment by kureta** [comment]
A global "skip silence" option also would be great.
